packages/analyticsdx-template-lint/src/schemas/template-info-schema.json

```
@@ -679,6 +679,30 @@
                 "body": "${0}"
               }
             ]
+          },
+          "liveConnection": {
+            "type": "object",
```

This should have a description, which shows as hover-text in the editor:
```suggestion
            "type": "object",
            "description": "The connection information for this live dataset"
```